packages/jsActions/nanoflow-actions-web/src/ToggleSidebar.ts

```diff
@@ -0,0 +1,22 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+import { Big } from "big.js";
+
+// BEGIN EXTRA CODE
+// END EXTRA CODE
+
+/**
+ * @returns {Promise.<void>}
+ */
+export async function ToggleSidebar() {
+    // BEGIN USER CODE
+    const sidebarToggleLocation = "com.mendix.widgets.web.sidebar.toggle";
+
+    (window as any)[sidebarToggleLocation]?.();
+    // END USER CODE
+}
```

Comment:
For my understanding, we don't need this nanoflow anymore for the "sidebar toggle button" implementation right?

Reply:
Yes, we do. We will have a button and JS action that can trigger the sidebar. The only thing that's needs to change in a follow up story (already covered by one of the Jira issues) is that the JS action needs to be combined with the one from native and put into the nanoflow commons package. The code will be separate though as native uses the mxui API and we cannot.

Reply:
👍